governance/program/src/state/proposal.rs

```diff
@@ -627,6 +644,23 @@ impl ProposalV2 {
         None
     }
 
+    /// Checks if vote can be tipped and transitioned to Vetoed state
+    /// If yes then Some(ProposalState::Vetoed) is returned and None otherwise
+    #[allow(clippy::float_cmp)]
```

Comment:
nit: are these needed? I don't see any float comparisons in these functions

Reply:
Good spot! removed